### ISSUE TITLE
typo in toolRoutes.js ?

### ISSUE THREAD

**Issue by keeferrourke** [ian-james/IFS#21]
On [line 16](https://github.com/ian-james/IFS/blob/master/ifs/app/components/Tool/toolRoutes.js#L16):
```
        if( req.session && !(req.session.toolSeelct && req.session.toolFile ) ) {
```
is `toolSeelct` supposed to be `toolSelect`?

**Comment by ian-james** [comment]
Yup definitely. Will change when I can

**Comment by ian-james** [comment]
Fixed as  of commit 017c4cb
